client/layout/masterbar/checkout.tsx

```diff
@@ -77,6 +86,13 @@ const CheckoutMasterbar: FunctionComponent< Props > = ( {
 				<span className="masterbar__secure-checkout-text">{ translate( 'Secure checkout' ) }</span>
 			</div>
 			<Item className="masterbar__item-title">{ title }</Item>
+			<Item
+				onClick={ () => setIsHelpCenterVisible( ! isHelpCenterVisible ) }
+				className={ classnames( 'masterbar__item-help', {
+					'is-active': isHelpCenterVisible,
+				} ) }
+				icon={ <HelpIcon newItems active={ isHelpCenterVisible } /> }
+			/>
```

Comment:
Almost forget! This should be behind a flag!

We can use this flag https://github.com/Automattic/wp-calypso/blob/update/help-center-checkout/config/development.json#L52

Reply:
Oh wow, great catch! Thanks

Reply:
Added the flag @alshakero